src/player.h

```diff
@@ -362,6 +362,7 @@ class Player
 	float hurt_tilt_timer;
 	float hurt_tilt_strength;
 
+	u16 protocol_version;
```

Comment:
Huh?? What is this for?  I can see that it's set on emerge, but it's not used anywhere.